### PR TITLE
Minor doc updates

### PR DESCRIPTION
**Pull request by chipzoller**
Minor doc updates, mostly JMESPath section updates. If accepting and merging, please squash so it's easier to cherry pick later.

### CODE REVIEW

**Comment by chipzoller** [comment]
@samj1912

File: content/en/docs/Writing policies/jmespath.md [hunk 1]
```diff
@@ -197,6 +195,16 @@ spec:
 
 To represent the `limits.memory` field in a JMESPath expression requires literal quoting of the key in order to avoid being interpreted as child nodes `limits` and `memory`. The expression would then be `{{ spec.hard.\"limits.memory\" }}`. A similar approach is needed when individual keys contain special characters, for example a dash (`-`). Quoting and then escaping is similarly needed there, ex., `{{ images.containers.\"my-container\".tag }}`.
 
+## Useful Patterns
+
+When developing policies for Kubernetes resources, there are several patterns which are common where JMESPath can be useful. This section attempts to capture example patterns that have been observed through real world use cases and how to write JMESPath for them.
+
+### Flattening Arrays
+
```
Comment (samj1912) [inline content/en/docs/Writing policies/jmespath.md:203]: Is this expected to be an empty section? (still WIP?)

**Review by samj1912 — APPROVED** [review]
Mostly lgtm. One minor comment.

File: content/en/docs/Writing policies/jmespath.md (hunk 1 above)
Comment (chipzoller) [reply content/en/docs/Writing policies/jmespath.md:203]: Yes lol

Comment (chipzoller) [reply content/en/docs/Writing policies/jmespath.md:203]: ...but now it's not.

**Comment by chipzoller** [comment]
Filled out the Flatten Array section. See what you think.

**Review by realshuting — APPROVED** [review]

File: content/en/docs/Writing policies/jmespath.md [hunk 2]
```diff
@@ -197,6 +195,118 @@ spec:
 
 To represent the `limits.memory` field in a JMESPath expression requires literal quoting of the key in order to avoid being interpreted as child nodes `limits` and `memory`. The expression would then be `{{ spec.hard.\"limits.memory\" }}`. A similar approach is needed when individual keys contain special characters, for example a dash (`-`). Quoting and then escaping is similarly needed there, ex., `{{ images.containers.\"my-container\".tag }}`.
 
+## Useful Patterns
+
+When developing policies for Kubernetes resources, there are several patterns which are common where JMESPath can be useful. This section attempts to capture example patterns that have been observed through real world use cases and how to write JMESPath for them.
+
+### Flattening Arrays
```
Comment (samj1912) [inline content/en/docs/Writing policies/jmespath.md:202]: Love this section. This presented use case is exactly what I was imagining when I looked at the section title!

**Review by samj1912 — APPROVED** [review]

**Comment by samj1912** [comment]
Ugh, sorry @chipzoller, clicked merge PR instead of squash. Let me help with the cherry pick.

**Comment by chipzoller** [comment]
Thanks!

**Pull request merged**